<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManage</class>
 <widget class="QMainWindow" name="UserManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="box1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>测试</string>
    </property>
    <widget class="QLineEdit" name="searchEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>381</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Kaiti Std</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OkBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_D"/>
   <addaction name="action_refresh"/>
   <addaction name="action_S"/>
   <addaction name="action_star"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_D"/>
    <addaction name="action_star"/>
    <addaction name="action_refresh"/>
    <addaction name="action_S"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_N">
   <property name="icon">
    <iconset>
     <normaloff>:/buttons/create.png</normaloff>:/buttons/create.png</iconset>
   </property>
   <property name="text">
    <string>添加(&amp;N)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="icon">
    <iconset>
     <normaloff>:/buttons/delete.png</normaloff>:/buttons/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="action_star">
   <property name="icon">
    <iconset>
     <normaloff>:/buttons/star1.png</normaloff>:/buttons/star1.png</iconset>
   </property>
   <property name="text">
    <string>会员变更</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="icon">
    <iconset>
     <normaloff>:/buttons/refresh.png</normaloff>:/buttons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset>
     <normaloff>:/buttons/save.png</normaloff>:/buttons/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
